app/experimenter/nimbus-ui/src/components/PageRequestReview/FormLaunchDraftToReview.tsx

```diff
@@ -36,7 +35,11 @@ const FormLaunchDraftToReview = ({
           <span className="text-danger">
             <AlertCircle /> We recommend previewing before launch.
           </span>{" "}
-          <a href="#" onClick={handleLaunchToPreviewClick}>
+          <a
+            href="#"
```

Comment:
ESLint [during compilation, not yarn lint:eslint (these two should really be one and the same grumble..)] is complaining that this needs to be a valid href: `The href attribute requires a valid value to be accessible.`. I think that means this could simply be `#launch` or similar. Or we could make it a button.